### ISSUE TITLE
[Accessibility] consider adding default keybinding to accept completion in inline accessible view

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#190716]
Type: <b>Feature Request</b>

In Accessible View (Alt+F2) for inline suggestion, users can activate some action buttons via keybindings, such as Alt+F6, Alt+[, Alt+]. However, "Accept Completion" does not have its default keybinding so users have to press Shift+Tab and hit Enter. Please add default keybinding for this, such as Ctrl+/ or Ctrl+Enter to accept the current suggestion.

VS Code version: Code - Insiders 1.82.0-insider (ccb95fd921349023027a0df25ed291b0992b9a18, 2023-08-17T05:33:29.141Z)
OS version: Windows_NT x64 10.0.22621
Modes:

**Comment by meganrogge** [comment]
I'm going to see if `Tab` will work here cc @rperez030, though I will go with your suggestions if not.

**Comment by jooyoungseo** [comment]
Personally, I think “tab” key is not a conceptually good option here because this is a separate and distinct view from the general suggestion widget so pressing tab key is more suitable for moving focus just like pressing Shift+Tab moves your focus to the action buttons. That’s why I suggest other keys, such as Ctrl+/ or Ctrl+Enter. Ctrl+/ is, for example, used in accepting copilot suggestion in a sidebar widget so this aligns with the existing key maps.

**Comment by meganrogge** [comment]
Thanks for your perspective here and I agree with it.

**Comment by meganrogge** [comment]
To verify, with screen reader mode enabled, trigger an inline completion. Use `alt+f2` to open the accessible view. Use `ctrl+/` to accept the completion.
